mods/translucent-windows.wh.cpp

```diff
@@ -849,6 +984,16 @@ void Wh_ModUninit(void)
 {
     g_settings.Unload = TRUE;
     ApplyForExistingWindows();
+
+    std::unordered_set<HWND> subclassedWindows;
+    {
+        std::lock_guard<std::mutex> guard(g_subclassedWindowsMutex);
+        subclassedWindows = std::move(g_subclassedWindows);
+        g_subclassedWindows.clear();
+    }
+
+    for (HWND hWnd : subclassedWindows)
+        UnsubclassWindow(hWnd);

```

Comment:
Since `g_subclassedWindows` is empty at this point, here it's enough to do:
```suggestion
        WindhawkUtils::RemoveWindowSubclassFromAnyThread(hWnd, SubclassProc);
```